lib/datasets.go

```diff
@@ -72,32 +71,29 @@ func (m DatasetMethods) Attributes() map[string]AttributeSet {
 
 // GetParams defines parameters for looking up the head or body of a dataset
 type GetParams struct {
+	params.List `json:"list,omitempty"`
 	// dataset reference to fetch; e.g. "b5/world_bank_population"
 	Ref string `json:"ref"`
 	// a component or nested field names to extract from the dataset; e.g. "body"
 	Selector string `json:"selector"`
-	// number of results to limit to. only applies when selector is "body"
-	Limit int `json:"limit"`
-	// number of results to skip. only applies when selector is "body"
-	Offset int `json:"offset"`
 	// TODO(dustmop): Remove `All` once `Cursor` is in use. Instead, callers should
 	// loop over their `Cursor` in order to get all rows.
+	// TODO(ramfox): are we in a place to remove All?
```

Comment:
Have we done enough with `Cursor` to remove the `All` field?

Reply:
It is okay to remove for `collections.List`, but not `Get`. The `dataset.Get` method doesn't return a cursor yet, and `cmd` isn't trying to consume a cursor.